healthScore/healthConfig/settings.py

```diff
@@ -113,7 +113,7 @@
 
 LANGUAGE_CODE = 'en-us'
 
-TIME_ZONE = 'UTC'
+TIME_ZONE = 'EST'
```

Comment:
We could maybe leave the timezone as UTC. That way, even NYU Shanghai/Abu Dhabi students would be able to access it with any timestamp issues

Reply:
Sounds good!

Reply:
Fixed